app/routes/identifikation/index.tsx

```diff
@@ -47,6 +51,18 @@ export const loader: LoaderFunction = async ({ request }) => {
   const dbUser = await findUserByEmail(sessionUser.email);
   if (!dbUser) return logoutDeletedUser(request);
 
+  const bundesIdentIsOnline =
+    !flags.isBundesIdentDisabled() && !flags.isBundesIdentDown();
+  const session = await getSession(request.headers.get("Cookie"));
+  const hasPrimaryOptionShown = Boolean(session.get("hasPrimaryOptionShown"));
+  if (
+    bundesIdentIsOnline &&
+    !hasPrimaryOptionShown &&
+    needsToStartIdentification(dbUser)
```

Comment:
I am wondering whether it would make sense to move this into a function `showBundesidentPrimayOptionPage` for more readability. Wdyt?